### PR TITLE
Merge 2.x

### PR DESCRIPTION
**Pull request by greg0ire**
Manual merge

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -33,22 +34,23 @@ env:
...
   include:
-    - php: '7.0'
+    - php: '7.1'
       env: TARGET=docs
-    - php: '7.0'
+    - php: '7.1'
       env: TARGET=lint
     - php: '5.6'
       env: COMPOSER_FLAGS="--prefer-lowest"
-    - php: '7.0'
```
Comment (jordisala1991) [inline .travis.yml]: there is somthing wrong here

**Comment by OskarStark** [comment]
Damn, a second to late..

**Comment by jordisala1991** [comment]
Dev-kit can fix it tonight

**Comment by jordisala1991** [comment]
it is only missing the `--prefer-lowest`

**Pull request merged**